conda.recipe/meta.yaml

```diff
@@ -1,34 +1,69 @@
 {% set data = load_setup_py_data() %}
 
 package:
-    name: weldx
-    version: {{ data["version"] }}
+  name: weldx
+  version: {{ data["version"] }}
 
 source:
     path: ..
 
 build:
-  script: pip install . -v
   number: 0
   noarch: python
+  script: pip install . -v
 
 requirements:
-    host:
-        - python >=3.8
-        - pip
-        - setuptools_scm
-    run:
-        - python >=3.8
-        # dependencies are defined in setup.py
-        {% for dep in data['install_requires'] %}
-        - {{ dep.lower() }}
-        {% endfor %}
+  host:
+    - pip
+    - setuptools_scm
+    - python >=3.8
+  run:
+    - python >=3.8
+    # dependencies are defined in setup.py
+    {% for dep in data['install_requires'] %}
+    - {{ dep.lower() }}
+    {% endfor %}
 
 test:
-    imports:
-        - weldx
-        - weldx.visualization
+  requires:
+    - pytest
+    - pytest-xdist
+  imports:
+    - weldx
+    - weldx.visualization
+  commands:
+    - pytest --pyargs weldx.tests -n auto
```

Comment:
this looks much cleaner now after #323 👍

Reply:
yes, indeed - very simple :)
And we are testing the packaged code, not the pointer to the working copy, where everything is always in the right spot. This is another benefit.